text/0000-icons.md

```diff
@@ -0,0 +1,137 @@
+- Start Date: 2021-03-14
+- RFC PR:
+- TVA Issue:
+
+# Summary
+
+Bundle icons as inline SVG into a simple, organized structure, ready for import and use in an application.
+
+Icons are formatted and optimized by automated build scripts.
+
+By default, the icon's color is inherited from the `color` property of the parent element.
+
+# Basic example
+
+## React (JSX)
+
+Import SVG as JSX element (default) from the library and use in your application.
+
+```jsx
+import { bookmark } from '@pluralsight/icons'
+
+const BookmarkIcon = () => <span style={{
+  display: "inline-block",
+  width: "14px",
+  height: "14px",
+  color: "#F00"
+}}>{bookmark}</span>
+```
+
+# Motivation
+
+To provide a single library for all of our icons in a simple, flexible format that is also accessible and performant.
```

Comment:
Not enough information here. This file should be a very detailed presentation vs having one-liners. Please use the same format that I have with including the questions in **bold** that are leading the discussion for the section.